.github/workflows/container-images-cd.yml

```diff
@@ -144,3 +144,21 @@ jobs:
                         "image_tag": "latest",
                         "context": ${{ toJson(github) }}
                       }
+
+            - name: Check for changes in posthog directory
+              id: check_changes_plugins
+              run: |
+                echo "::set-output name=changed::$(git diff --name-only HEAD^ HEAD | grep '^plugin-server/' || true)"
+
+            - name: Trigger Ingestion Cloud deployment
+              if: steps.check_changes_plugins.outputs.changed != ''
+              uses: peter-evans/repository-dispatch@v1
+              with:
+                  token: ${{ secrets.GITHUB_TOKEN }}
+                  repository: PostHog/charts
+                  event-type: ingestion_deploy
+                  client-payload: |
+                      {
+                        "image_tag": "latest",
```

Comment:
I'm not sure this will actually trigger a rollout every time, as going from latest -> latest will be a no-op when helm runs a diff

It's probably best if we actually reference the tag for the image, so it diffs and rolls out properly

Reply:
💯